nginx-shibboleth/config.sls

```diff
@@ -1,15 +1,33 @@
 {% from "nginx-shibboleth/map.jinja" import nginx_shibboleth with context %}
+{% set pillar_config = salt.pillar.get('nginx-shibboleth:config', {}) %}
 
 include:
   - .install
   - .service
 
-nginx-shibboleth-config:
+{% if nginx_shibboleth.generate_sp_cert %}
+generate_shibboleth_sp_cert_and_key:
+  cmd.run:
+    - name: shib-keygen
+    - creates: /etc/shibboleth/sp-cert.pem
+{% else %}
+{% for secret in ['key', 'cert'] %}
```

Comment:
How is that finding ['key', 'cert'] since in the [pillar.example](https://github.com/mitodl/nginx-shibboleth-formula/blob/blarghmatey/shibboleth_install/pillar.example#L5) those are under `secrets`?

Reply:
Those are just string variables used for interpolation in  the file.managed states.